js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_pt.properties

```diff
@@ -0,0 +1,188 @@
+confirm=Confirmar
+updateSuccess=O recurso foi atualizado com sucesso.
+selectOne=Selecione uma opção
+unShare=Descompartilhar tudo
+linkedAccounts=Contas conectadas
+personalInfoDescription=Gerir as suas informações básicas
+removeAccess=Remover Acesso
+signingInDescription=Configure maneiras de fazer login.
+somethingWentWrongDescription=Desculpe, ocorreu um erro inesperado.
+personalInfo=Informação pessoal
+removeCred=Remover {{name}}
+signOutAllDevices=Terminar a sessão em todos os dispositivos
+scopeNameHelp=Nome do escopo do cliente. Deve ser único no realm. O nome não deve conter espaços, pois é usado como valor do parâmetro de escopo
+flowTypeHelp=Que tipo de forma é essa
+error-invalid-email=Endereço de e-mail inválido.
+myResources=Meus Recursos
+linkedAccountsIntroMessage=Gerir acessos por conta de terceiros.
+accountSecurity=Segurança da conta
+groupDescriptionLabel=Veja os grupos aos quais está associado
+manageAccount=Gerir conta
+resourceAlreadyShared=O recurso já foi compartilhado com este utilizador.
+error-invalid-length-too-long='{{0}}' deve ter tamanho máximo de {{2}}.
+accountUpdatedMessage=A sua conta foi atualizada.
+shareError=Não foi possível compartilhar o recurso devido a\: {{error}}
+logo=Logotipo
+password-help-text=Entre com a sua palavra-passe.
+deleteAccount=Apagar Conta
+permissionRequest=Pedidos de permissão - {{name}}
+add=Adicionar
+error-invalid-value='{{0}}' tem valor inválido.
+somethingWentWrong=Houve um erro algures
+rolesScope=Se não houver mapeamento de escopo de função definido, cada utilizador terá permissão para usar esse escopo de cliente. Se houver mapeamentos de escopo de função definidos, o utilizador deverá ser membro de pelo menos uma das funções.
+unShareError=Não foi possível cancelar o compartilhamento do recurso devido a\: {{error}}
+ipAddress=Endereço IP
+tryAgain=Tente novamente
+resourceName=Nome do Recurso
+unlinkedEmpty=Nenhum Provedor Não-Conectado
+done=Pronto
+basic-authentication=Autenticação Básica
+socialLogin=Login Social
+notInUse=Não utilizado
+unShareAllConfirm=Tem certeza de que quer remover todos os compartilhamentos?
+firstName=Primeiro nome
+name=Nome
+clientTypeHelp=O tipo deste recurso. Pode ser usado para agrupar diferentes instâncias de recursos com o mesmo tipo.
+unknownUser=Anónimo
+offlineAccess=Acesso Offline
+password-display-name=Palavra-passe
+expires=Expira em
+noGroups=Nenhum grupo
+notSetUp={{name}} não está configurada.
+error-invalid-number='{{0}}' é um número inválido.
+link=Conectar Conta
+shareTheResource=Compartilhe o recurso - {{name}}
+description=Descrição
+update=Atualizar
+choose=Escolha...
+signedInDevices=Dispositivos autenticados
+delete=Apagar
+missingPasswordMessage='{{0}}'\: Por favor, insira a palavra-passe.
+otp-help-text=Insira o código de verificação da aplicação autenticador.
+groupsListColumnsNames=Nomes de colunas de lista de grupos
+error-invalid-uri-scheme='{{0}}' tem esquema de URL inválido.
```

Comment:
Não falta aqui um "um" ou "o" as in "tem o esquema de URL inválido"

Reply:
ficou `error-invalid-uri-scheme='{{0}}' é um esquema de URL inválido.`